src/app/user-service/persist-storage.ts

```diff
@@ -69,9 +70,9 @@ export class PersistStorage {
     private _itemChange = new Subject<{key: string, value: string}>();
 
     constructor(private _userService: UserService) {
-        this._userService.getUserInfo()
+        this._userService.userInfo
             .subscribe((user) => {
-                if (!user) {
+                if (!user || user.id === User.ID_INITIAL_USER) {
```

Comment:
Why is `user.id === User.ID_INITIAL_USER` added?

Reply:
I modified this code to prevent unwanted storage clear when a user opens the app just because the first user.id will always be the ID_INITIAL_USER as the default value.

It now should just do the clear when the previous user is not initial value.